faq/README.md

```diff
@@ -137,12 +137,14 @@ This is usually an issue with AMD's graphics driver. Make sure you disable AMD C
 
 ### My overlay doesn't sort by DPS
 
-If you're using the ngld OverlayPlugin, you have to enable sorting. Go to `Plugins` > `MiniParseEventSource` and set `Sort By` to `DPS`.
+You have to enable sorting. Go to `Plugins` > `MiniParseEventSource` and set `Sort By` to `DPS`.
 
-### ngld OverlayPlugin failed because it couldn't download something
+### OverlayPlugin failed because it couldn't download something
 
 Go to the OverlayPlugin tab. It'll tell you more about the file it tried to download. You'll be able to retry the download or manually download the required file there.
 
+If you're still having issues updating, make sure you are using the new OverlayPlugin fork, and not the ngld or hibiyasleep version.
```

Comment:
I don't think this is a relevant answer to failed downloads.  Only ngld/new fork do downloads, and the ngld version should still be working here.

Reply:
That might be technically true, but do we want to have to worry about supporting people using old versions? I think it's only going to cause problems.

Reply:
I don't understand what you are trying to add here.  Is there confusion between downloads (of updates) and downloads (of CEF).  This FAQ entry is about the latter (CEF) and still applies to the current fork.  Additionally, there's no issues with the update in older versions, it just says there are no new updates.

Reply:
No, I understand it. I'm saying that there's no harm in making sure the user is on the latest version, and not really any benefit of the user staying on an unmaintained version. To avoid unnecessary support questions (even if it theoretically still works on the old version), I'd rather just push the user to use the maintained version). 

I can remove it if it seems that out of place, but I do feel like "make sure you're on the latest version" is kind of a universal question when someone asks for support, so might as well pre-empt it if possible.

Reply:
I just don't think it's relevant to this particular issue.  Downloads haven't failed because of a different fork ever?  I think if we want to broadcast "be on the later version" it'd be better done as a "what version should I be on" question at the top and not buried in some entirely unrelated answer.